db/migrate/20190413044706_add_lastname_to_volunteers.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastnameToVolunteers < ActiveRecord::Migration[5.1]
+  def change
+    add_column :volunteers, :lastname, :string
```

Comment:
In general your migrations are a bit of a mess - might be better to revert to an older migration and then try to condense this into a few migrations.